src/main/java/org/distributed/systems/chord/actors/MemCachedHandler.java

```diff
@@ -0,0 +1,73 @@
+package org.distributed.systems.chord.actors;
+
+import akka.actor.AbstractActor;
+import akka.io.Tcp;
+import akka.io.TcpMessage;
+import akka.util.ByteString;
+import org.distributed.systems.chord.service.StorageService;
+
+import java.io.Serializable;
+import java.util.Arrays;
+
+class MemCachedHandler extends AbstractActor {
```

Comment:
Nice that we already have this!!! 👍

Reply:
It's very basic though 😅 The MemCache specification is much longer. Should check what we need to implement for a respresentative benchmark.